website/content/docs/install/index.mdx

```diff
@@ -322,6 +334,35 @@ net.bridge.bridge-nf-call-iptables = 1
 
 </CodeBlockConfig>
 
+<h3>Install `consul-cni` for Consul service mesh</h3>
+
+If the `transparent_proxy` block is used for Consul Service Mesh, then the 
+`consul-cni` CNI Plugin must be installed on each client node to redirect inbound 
+and outbound traffic for services to the Envoy proxy. The following commands
+assume that the linux packages repositories have already been added using `apt` or 
+`yum` and that the reference CNI plugins have already been installed. 
+
+<Tabs>
+
+<Tab heading="CentOS/RHEL">
+
+Install the `consul-cni` plugin on each client node.  
+
+```shell-session
+$ sudo yum -y install consul-cni
+```
+</Tab>
+<Tab heading="Ubuntu/Debian">
+
+Install the `consul-cni` plugin on each client node.  
+
+```shell-session
+$ sudo apt-get update && sudo apt-get install consul-cni
```

Comment:
If you've done the installation steps above to set up the repo, you don't need to update because you'll already have the package list. If you haven't done the steps above, the update won't get you the package anyways.

Also, we should probably use `-y` here if we're going to use it for CentOS/RHEL too.

```suggestion
$ sudo apt-get install -y consul-cni
```